src/elements/autocomplete/autocomplete-base-element.ts

```diff
@@ -242,7 +241,7 @@ abstract class SbbAutocompleteBaseElement<T = string> extends SbbNegativeMixin(
     if (isServer || !this.hasUpdated) {
       return;
     }
-    if (!name || name === 'trigger') {
+    if ((!name || name === 'trigger') && this.hasUpdated) {
```

Comment:
One line above it is returning if not updated, so this check is probably redundant